app/controllers/users_controller.rb

```diff
@@ -0,0 +1,42 @@
+class UsersController < ApplicationController
+   before_action only: [:edit, :index, :create]
+  def index
+    @user = User.all
+  end
+  def new
```

Comment:
```suggestion
  end
  
  def new
```
各定義(defの前やendの後)には改行を入れてあげましょう！
他箇所では改行を入れているところもあるのですが、コードの書き方を統一するのは地味ですがとても大事です！